pydantic/fields.py

```diff
@@ -393,7 +399,32 @@ def Field(
     :param json_schema_extra: extra dict to be merged with the JSON Schema for this field
     :param strict: enable or disable strict parsing mode
     :param validate_default: whether the default value should be validated for this field
+    :param const: removed, use ``Literal`` instead'
+    :param unique_items: removed, use ``Set`` instead
+    :param allow_mutation: deprecated, use ``frozen`` instead
+    :param regex: removed, use `Pattern` instead
+    :param **extra: removed
     """
+    # Check deprecated & removed params of V1.
+    # This has to be removed deprecation period over.
+    if const:
+        raise ValueError('`const` is removed. use `Literal` instead')
```

Comment:
Please use `PydanticUserError`, add a code and a section to docs if you think necessary, otherwise set `code=None`.

Reply:
Done